commit df8501c96cf35eac586cce81c5547060ef22f624
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 11:18:31 2010 -0700

    Staging: brcm80211: remove CONST definition
    
    Use 'const', it's the correct thing to do.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/bcmdefs.h
+++ b/drivers/staging/brcm80211/include/bcmdefs.h
@@ -56,7 +56,6 @@
 #define BCMOVERLAY1DATA(_sym)	_sym
 #define BCMOVERLAY1FN(_fn)	_fn
 #define BCMOVERLAYERRFN(_fn)	_fn
-#define CONST	const
 #ifdef mips
 #define BCMFASTPATH		__attribute__ ((__section__(".text.fastpath")))
 #else
--- a/drivers/staging/brcm80211/phy/wlc_phy_lcn.c
+++ b/drivers/staging/brcm80211/phy/wlc_phy_lcn.c
@@ -544,11 +544,11 @@ static const s8 lcnphy_gain_index_offset
 	-2
 };
 
-extern CONST u8 spur_tbl_rev0[];
-extern CONST uint32 dot11lcnphytbl_rx_gain_info_sz_rev1;
-extern CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_rev1[];
-extern CONST dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_bt_epa;
-extern CONST dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_bt_epa_p250;
+extern const u8 spur_tbl_rev0[];
+extern const uint32 dot11lcnphytbl_rx_gain_info_sz_rev1;
+extern const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_rev1[];
+extern const dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_bt_epa;
+extern const dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_bt_epa_p250;
 
 typedef struct _chan_info_2064_lcnphy {
 	uint chan;
@@ -1049,7 +1049,7 @@ void wlc_lcnphy_read_table(phy_info_t *p
 
 static void
 wlc_lcnphy_common_read_table(phy_info_t *pi, uint32 tbl_id,
-			     CONST void *tbl_ptr, uint32 tbl_len,
+			     const void *tbl_ptr, uint32 tbl_len,
 			     uint32 tbl_width, uint32 tbl_offset)
 {
 	phytbl_info_t tab;
@@ -1063,7 +1063,7 @@ wlc_lcnphy_common_read_table(phy_info_t
 
 static void
 wlc_lcnphy_common_write_table(phy_info_t *pi, uint32 tbl_id,
-			      CONST void *tbl_ptr, uint32 tbl_len,
+			      const void *tbl_ptr, uint32 tbl_len,
 			      uint32 tbl_width, uint32 tbl_offset)
 {
 
@@ -1975,13 +1975,13 @@ wlc_lcnphy_tx_iqlo_cal(phy_info_t *pi,
 	write_phy_reg(pi, 0x93d, 0xc0);
 
 	wlc_lcnphy_common_write_table(pi, LCNPHY_TBL_ID_IQLOCAL,
-				      (CONST void *)
+				      (const void *)
 				      lcnphy_iqcal_loft_gainladder,
 				      ARRAYSIZE(lcnphy_iqcal_loft_gainladder),
 				      16, 0);
 
 	wlc_lcnphy_common_write_table(pi, LCNPHY_TBL_ID_IQLOCAL,
-				      (CONST void *)lcnphy_iqcal_ir_gainladder,
+				      (const void *)lcnphy_iqcal_ir_gainladder,
 				      ARRAYSIZE(lcnphy_iqcal_ir_gainladder), 16,
 				      32);
 
--- a/drivers/staging/brcm80211/phy/wlc_phytbl_lcn.c
+++ b/drivers/staging/brcm80211/phy/wlc_phytbl_lcn.c
@@ -19,7 +19,7 @@
 #include <wlc_phy_int.h>
 #include <wlc_phytbl_lcn.h>
 
-CONST uint32 dot11lcn_gain_tbl_rev0[] = {
+const uint32 dot11lcn_gain_tbl_rev0[] = {
 	0x00000000,
 	0x00000000,
 	0x00000000,
@@ -118,7 +118,7 @@ CONST uint32 dot11lcn_gain_tbl_rev0[] =
 	0x00000000,
 };
 
-CONST uint32 dot11lcn_gain_tbl_rev1[] = {
+const uint32 dot11lcn_gain_tbl_rev1[] = {
 	0x00000000,
 	0x00000000,
 	0x00000000,
@@ -217,7 +217,7 @@ CONST uint32 dot11lcn_gain_tbl_rev1[] =
 	0x00000000,
 };
 
-CONST uint16 dot11lcn_aux_gain_idx_tbl_rev0[] = {
+const uint16 dot11lcn_aux_gain_idx_tbl_rev0[] = {
 	0x0401,
 	0x0402,
 	0x0403,
@@ -258,7 +258,7 @@ CONST uint16 dot11lcn_aux_gain_idx_tbl_r
 	0x0000,
 };
 
-CONST uint32 dot11lcn_gain_idx_tbl_rev0[] = {
+const uint32 dot11lcn_gain_idx_tbl_rev0[] = {
 	0x00000000,
 	0x00000000,
 	0x10000000,
@@ -409,7 +409,7 @@ CONST uint32 dot11lcn_gain_idx_tbl_rev0[
 	0x0000001c,
 };
 
-CONST uint16 dot11lcn_aux_gain_idx_tbl_2G[] = {
+const uint16 dot11lcn_aux_gain_idx_tbl_2G[] = {
 	0x0000,
 	0x0000,
 	0x0000,
@@ -450,7 +450,7 @@ CONST uint16 dot11lcn_aux_gain_idx_tbl_2
 	0x0000
 };
 
-CONST u8 dot11lcn_gain_val_tbl_2G[] = {
+const u8 dot11lcn_gain_val_tbl_2G[] = {
 	0xfc,
 	0x02,
 	0x08,
@@ -521,7 +521,7 @@ CONST u8 dot11lcn_gain_val_tbl_2G[] = {
 	0x00
 };
 
-CONST uint32 dot11lcn_gain_idx_tbl_2G[] = {
+const uint32 dot11lcn_gain_idx_tbl_2G[] = {
 	0x00000000,
 	0x00000000,
 	0x00000000,
@@ -676,7 +676,7 @@ CONST uint32 dot11lcn_gain_idx_tbl_2G[]
 	0x00000000
 };
 
-CONST uint32 dot11lcn_gain_tbl_2G[] = {
+const uint32 dot11lcn_gain_tbl_2G[] = {
 	0x00000000,
 	0x00000004,
 	0x00000008,
@@ -775,7 +775,7 @@ CONST uint32 dot11lcn_gain_tbl_2G[] = {
 	0x00000000
 };
 
-CONST uint32 dot11lcn_gain_tbl_extlna_2G[] = {
+const uint32 dot11lcn_gain_tbl_extlna_2G[] = {
 	0x00000000,
 	0x00000004,
 	0x00000008,
@@ -874,7 +874,7 @@ CONST uint32 dot11lcn_gain_tbl_extlna_2G
 	0x00000000
 };
 
-CONST uint16 dot11lcn_aux_gain_idx_tbl_extlna_2G[] = {
+const uint16 dot11lcn_aux_gain_idx_tbl_extlna_2G[] = {
 	0x0400,
 	0x0400,
 	0x0400,
@@ -915,7 +915,7 @@ CONST uint16 dot11lcn_aux_gain_idx_tbl_e
 	0x0000
 };
 
-CONST u8 dot11lcn_gain_val_tbl_extlna_2G[] = {
+const u8 dot11lcn_gain_val_tbl_extlna_2G[] = {
 	0xfc,
 	0x02,
 	0x08,
@@ -986,7 +986,7 @@ CONST u8 dot11lcn_gain_val_tbl_extlna_2G
 	0x00
 };
 
-CONST uint32 dot11lcn_gain_idx_tbl_extlna_2G[] = {
+const uint32 dot11lcn_gain_idx_tbl_extlna_2G[] = {
 	0x00000000,
 	0x00000040,
 	0x00000000,
@@ -1141,7 +1141,7 @@ CONST uint32 dot11lcn_gain_idx_tbl_extln
 	0x00000000
 };
 
-CONST uint32 dot11lcn_aux_gain_idx_tbl_5G[] = {
+const uint32 dot11lcn_aux_gain_idx_tbl_5G[] = {
 	0x0000,
 	0x0000,
 	0x0000,
@@ -1182,7 +1182,7 @@ CONST uint32 dot11lcn_aux_gain_idx_tbl_5
 	0x0000
 };
 
-CONST uint32 dot11lcn_gain_val_tbl_5G[] = {
+const uint32 dot11lcn_gain_val_tbl_5G[] = {
 	0xf7,
 	0xfd,
 	0x00,
@@ -1253,7 +1253,7 @@ CONST uint32 dot11lcn_gain_val_tbl_5G[]
 	0x00
 };
 
-CONST uint32 dot11lcn_gain_idx_tbl_5G[] = {
+const uint32 dot11lcn_gain_idx_tbl_5G[] = {
 	0x00000000,
 	0x00000000,
 	0x00000000,
@@ -1408,7 +1408,7 @@ CONST uint32 dot11lcn_gain_idx_tbl_5G[]
 	0x00000000
 };
 
-CONST uint32 dot11lcn_gain_tbl_5G[] = {
+const uint32 dot11lcn_gain_tbl_5G[] = {
 	0x00000000,
 	0x00000040,
 	0x00000080,
@@ -1507,7 +1507,7 @@ CONST uint32 dot11lcn_gain_tbl_5G[] = {
 	0x00000000
 };
 
-CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_rev0[] = {
+const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_rev0[] = {
 	{&dot11lcn_gain_tbl_rev0,
 	 sizeof(dot11lcn_gain_tbl_rev0) / sizeof(dot11lcn_gain_tbl_rev0[0]), 18,
 	 0, 32}
@@ -1522,7 +1522,7 @@ CONST dot11lcnphytbl_info_t dot11lcnphyt
 	,
 };
 
-CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_rev1[] = {
+const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_rev1[] = {
 	{&dot11lcn_gain_tbl_rev1,
 	 sizeof(dot11lcn_gain_tbl_rev1) / sizeof(dot11lcn_gain_tbl_rev1[0]), 18,
 	 0, 32}
@@ -1537,7 +1537,7 @@ CONST dot11lcnphytbl_info_t dot11lcnphyt
 	,
 };
 
-CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_2G_rev2[] = {
+const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_2G_rev2[] = {
 	{&dot11lcn_gain_tbl_2G,
 	 sizeof(dot11lcn_gain_tbl_2G) / sizeof(dot11lcn_gain_tbl_2G[0]), 18, 0,
 	 32}
@@ -1555,7 +1555,7 @@ CONST dot11lcnphytbl_info_t dot11lcnphyt
 	 17, 0, 8}
 };
 
-CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_5G_rev2[] = {
+const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_5G_rev2[] = {
 	{&dot11lcn_gain_tbl_5G,
 	 sizeof(dot11lcn_gain_tbl_5G) / sizeof(dot11lcn_gain_tbl_5G[0]), 18, 0,
 	 32}
@@ -1573,7 +1573,7 @@ CONST dot11lcnphytbl_info_t dot11lcnphyt
 	 17, 0, 8}
 };
 
-CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_extlna_2G_rev2[] = {
+const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_extlna_2G_rev2[] = {
 	{&dot11lcn_gain_tbl_extlna_2G,
 	 sizeof(dot11lcn_gain_tbl_extlna_2G) /
 	 sizeof(dot11lcn_gain_tbl_extlna_2G[0]), 18, 0, 32}
@@ -1591,7 +1591,7 @@ CONST dot11lcnphytbl_info_t dot11lcnphyt
 	 sizeof(dot11lcn_gain_val_tbl_extlna_2G[0]), 17, 0, 8}
 };
 
-CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_extlna_5G_rev2[] = {
+const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_extlna_5G_rev2[] = {
 	{&dot11lcn_gain_tbl_5G,
 	 sizeof(dot11lcn_gain_tbl_5G) / sizeof(dot11lcn_gain_tbl_5G[0]), 18, 0,
 	 32}
@@ -1609,23 +1609,23 @@ CONST dot11lcnphytbl_info_t dot11lcnphyt
 	 17, 0, 8}
 };
 
-CONST uint32 dot11lcnphytbl_rx_gain_info_sz_rev0 =
+const uint32 dot11lcnphytbl_rx_gain_info_sz_rev0 =
     sizeof(dot11lcnphytbl_rx_gain_info_rev0) /
     sizeof(dot11lcnphytbl_rx_gain_info_rev0[0]);
 
-CONST uint32 dot11lcnphytbl_rx_gain_info_sz_rev1 =
+const uint32 dot11lcnphytbl_rx_gain_info_sz_rev1 =
     sizeof(dot11lcnphytbl_rx_gain_info_rev1) /
     sizeof(dot11lcnphytbl_rx_gain_info_rev1[0]);
 
-CONST uint32 dot11lcnphytbl_rx_gain_info_2G_rev2_sz =
+const uint32 dot11lcnphytbl_rx_gain_info_2G_rev2_sz =
     sizeof(dot11lcnphytbl_rx_gain_info_2G_rev2) /
     sizeof(dot11lcnphytbl_rx_gain_info_2G_rev2[0]);
 
-CONST uint32 dot11lcnphytbl_rx_gain_info_5G_rev2_sz =
+const uint32 dot11lcnphytbl_rx_gain_info_5G_rev2_sz =
     sizeof(dot11lcnphytbl_rx_gain_info_5G_rev2) /
     sizeof(dot11lcnphytbl_rx_gain_info_5G_rev2[0]);
 
-CONST uint16 dot11lcn_min_sig_sq_tbl_rev0[] = {
+const uint16 dot11lcn_min_sig_sq_tbl_rev0[] = {
 	0x014d,
 	0x014d,
 	0x014d,
@@ -1692,7 +1692,7 @@ CONST uint16 dot11lcn_min_sig_sq_tbl_rev
 	0x014d,
 };
 
-CONST uint16 dot11lcn_noise_scale_tbl_rev0[] = {
+const uint16 dot11lcn_noise_scale_tbl_rev0[] = {
 	0x0000,
 	0x0000,
 	0x0000,
@@ -1759,7 +1759,7 @@ CONST uint16 dot11lcn_noise_scale_tbl_re
 	0x0000,
 };
 
-CONST uint32 dot11lcn_fltr_ctrl_tbl_rev0[] = {
+const uint32 dot11lcn_fltr_ctrl_tbl_rev0[] = {
 	0x000141f8,
 	0x000021f8,
 	0x000021fb,
@@ -1772,7 +1772,7 @@ CONST uint32 dot11lcn_fltr_ctrl_tbl_rev0
 	0x0000024b,
 };
 
-CONST uint32 dot11lcn_ps_ctrl_tbl_rev0[] = {
+const uint32 dot11lcn_ps_ctrl_tbl_rev0[] = {
 	0x00100001,
 	0x00200010,
 	0x00300001,
@@ -1795,7 +1795,7 @@ CONST uint32 dot11lcn_ps_ctrl_tbl_rev0[]
 	0x00600f22,
 };
 
-CONST uint16 dot11lcn_sw_ctrl_tbl_4313_epa_rev0_combo[] = {
+const uint16 dot11lcn_sw_ctrl_tbl_4313_epa_rev0_combo[] = {
 	0x0007,
 	0x0005,
 	0x0006,
@@ -1863,7 +1863,7 @@ CONST uint16 dot11lcn_sw_ctrl_tbl_4313_e
 
 };
 
-CONST uint16 dot11lcn_sw_ctrl_tbl_4313_bt_epa_p250_rev0[] = {
+const uint16 dot11lcn_sw_ctrl_tbl_4313_bt_epa_p250_rev0[] = {
 	0x0007,
 	0x0005,
 	0x0002,
@@ -1930,7 +1930,7 @@ CONST uint16 dot11lcn_sw_ctrl_tbl_4313_b
 	0x0002,
 };
 
-CONST uint16 dot11lcn_sw_ctrl_tbl_4313_epa_rev0[] = {
+const uint16 dot11lcn_sw_ctrl_tbl_4313_epa_rev0[] = {
 	0x0002,
 	0x0008,
 	0x0004,
@@ -1997,7 +1997,7 @@ CONST uint16 dot11lcn_sw_ctrl_tbl_4313_e
 	0x0001,
 };
 
-CONST uint16 dot11lcn_sw_ctrl_tbl_4313_rev0[] = {
+const uint16 dot11lcn_sw_ctrl_tbl_4313_rev0[] = {
 	0x000a,
 	0x0009,
 	0x0006,
@@ -2064,7 +2064,7 @@ CONST uint16 dot11lcn_sw_ctrl_tbl_4313_r
 	0x0005,
 };
 
-CONST uint16 dot11lcn_sw_ctrl_tbl_rev0[] = {
+const uint16 dot11lcn_sw_ctrl_tbl_rev0[] = {
 	0x0004,
 	0x0004,
 	0x0002,
@@ -2131,7 +2131,7 @@ CONST uint16 dot11lcn_sw_ctrl_tbl_rev0[]
 	0x0002,
 };
 
-CONST u8 dot11lcn_nf_table_rev0[] = {
+const u8 dot11lcn_nf_table_rev0[] = {
 	0x5f,
 	0x36,
 	0x29,
@@ -2150,7 +2150,7 @@ CONST u8 dot11lcn_nf_table_rev0[] = {
 	0x1f,
 };
 
-CONST u8 dot11lcn_gain_val_tbl_rev0[] = {
+const u8 dot11lcn_gain_val_tbl_rev0[] = {
 	0x09,
 	0x0f,
 	0x14,
@@ -2213,7 +2213,7 @@ CONST u8 dot11lcn_gain_val_tbl_rev0[] =
 	0x00,
 };
 
-CONST u8 dot11lcn_spur_tbl_rev0[] = {
+const u8 dot11lcn_spur_tbl_rev0[] = {
 	0x01,
 	0x01,
 	0x01,
@@ -2344,7 +2344,7 @@ CONST u8 dot11lcn_spur_tbl_rev0[] = {
 	0x01,
 };
 
-CONST uint16 dot11lcn_unsup_mcs_tbl_rev0[] = {
+const uint16 dot11lcn_unsup_mcs_tbl_rev0[] = {
 	0x001a,
 	0x0034,
 	0x004e,
@@ -2501,7 +2501,7 @@ CONST uint16 dot11lcn_unsup_mcs_tbl_rev0
 	0x06f6,
 };
 
-CONST uint16 dot11lcn_iq_local_tbl_rev0[] = {
+const uint16 dot11lcn_iq_local_tbl_rev0[] = {
 	0x0200,
 	0x0300,
 	0x0400,
@@ -2612,7 +2612,7 @@ CONST uint16 dot11lcn_iq_local_tbl_rev0[
 	0x0000,
 };
 
-CONST uint32 dot11lcn_papd_compdelta_tbl_rev0[] = {
+const uint32 dot11lcn_papd_compdelta_tbl_rev0[] = {
 	0x00080000,
 	0x00080000,
 	0x00080000,
@@ -2775,7 +2775,7 @@ CONST uint32 dot11lcn_papd_compdelta_tbl
 	0x00080000,
 };
 
-CONST dot11lcnphytbl_info_t dot11lcnphytbl_info_rev0[] = {
+const dot11lcnphytbl_info_t dot11lcnphytbl_info_rev0[] = {
 	{&dot11lcn_min_sig_sq_tbl_rev0,
 	 sizeof(dot11lcn_min_sig_sq_tbl_rev0) /
 	 sizeof(dot11lcn_min_sig_sq_tbl_rev0[0]), 2, 0, 16}
@@ -2834,34 +2834,34 @@ CONST dot11lcnphytbl_info_t dot11lcnphyt
 	,
 };
 
-CONST dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313 = {
+const dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313 = {
 	&dot11lcn_sw_ctrl_tbl_4313_rev0,
 	    sizeof(dot11lcn_sw_ctrl_tbl_4313_rev0) /
 	    sizeof(dot11lcn_sw_ctrl_tbl_4313_rev0[0]), 15, 0, 16
 };
 
-CONST dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_epa = {
+const dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_epa = {
 	&dot11lcn_sw_ctrl_tbl_4313_epa_rev0,
 	    sizeof(dot11lcn_sw_ctrl_tbl_4313_epa_rev0) /
 	    sizeof(dot11lcn_sw_ctrl_tbl_4313_epa_rev0[0]), 15, 0, 16
 };
 
-CONST dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_bt_epa = {
+const dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_bt_epa = {
 	&dot11lcn_sw_ctrl_tbl_4313_epa_rev0_combo,
 	    sizeof(dot11lcn_sw_ctrl_tbl_4313_epa_rev0_combo) /
 	    sizeof(dot11lcn_sw_ctrl_tbl_4313_epa_rev0_combo[0]), 15, 0, 16
 };
 
-CONST dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_bt_epa_p250 = {
+const dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_bt_epa_p250 = {
 	&dot11lcn_sw_ctrl_tbl_4313_bt_epa_p250_rev0,
 	    sizeof(dot11lcn_sw_ctrl_tbl_4313_bt_epa_p250_rev0) /
 	    sizeof(dot11lcn_sw_ctrl_tbl_4313_bt_epa_p250_rev0[0]), 15, 0, 16
 };
 
-CONST uint32 dot11lcnphytbl_info_sz_rev0 =
+const uint32 dot11lcnphytbl_info_sz_rev0 =
     sizeof(dot11lcnphytbl_info_rev0) / sizeof(dot11lcnphytbl_info_rev0[0]);
 
-CONST lcnphy_tx_gain_tbl_entry dot11lcnphy_2GHz_extPA_gaintable_rev0[128] = {
+const lcnphy_tx_gain_tbl_entry dot11lcnphy_2GHz_extPA_gaintable_rev0[128] = {
 	{3, 0, 31, 0, 72,}
 	,
 	{3, 0, 31, 0, 70,}
@@ -3120,7 +3120,7 @@ CONST lcnphy_tx_gain_tbl_entry dot11lcnp
 	,
 };
 
-CONST lcnphy_tx_gain_tbl_entry dot11lcnphy_2GHz_gaintable_rev0[128] = {
+const lcnphy_tx_gain_tbl_entry dot11lcnphy_2GHz_gaintable_rev0[128] = {
 	{7, 0, 31, 0, 72,}
 	,
 	{7, 0, 31, 0, 70,}
@@ -3379,7 +3379,7 @@ CONST lcnphy_tx_gain_tbl_entry dot11lcnp
 	,
 };
 
-CONST lcnphy_tx_gain_tbl_entry dot11lcnphy_5GHz_gaintable_rev0[128] = {
+const lcnphy_tx_gain_tbl_entry dot11lcnphy_5GHz_gaintable_rev0[128] = {
 	{255, 255, 0xf0, 0, 152,}
 	,
 	{255, 255, 0xf0, 0, 147,}
--- a/drivers/staging/brcm80211/phy/wlc_phytbl_lcn.h
+++ b/drivers/staging/brcm80211/phy/wlc_phytbl_lcn.h
@@ -16,24 +16,24 @@
 
 typedef phytbl_info_t dot11lcnphytbl_info_t;
 
-extern CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_rev0[];
-extern CONST uint32 dot11lcnphytbl_rx_gain_info_sz_rev0;
-extern CONST dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313;
-extern CONST dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_epa;
-extern CONST dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_epa_combo;
+extern const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_rev0[];
+extern const uint32 dot11lcnphytbl_rx_gain_info_sz_rev0;
+extern const dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313;
+extern const dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_epa;
+extern const dot11lcnphytbl_info_t dot11lcn_sw_ctrl_tbl_info_4313_epa_combo;
 
-extern CONST dot11lcnphytbl_info_t dot11lcnphytbl_info_rev0[];
-extern CONST uint32 dot11lcnphytbl_info_sz_rev0;
+extern const dot11lcnphytbl_info_t dot11lcnphytbl_info_rev0[];
+extern const uint32 dot11lcnphytbl_info_sz_rev0;
 
-extern CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_2G_rev2[];
-extern CONST uint32 dot11lcnphytbl_rx_gain_info_2G_rev2_sz;
+extern const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_2G_rev2[];
+extern const uint32 dot11lcnphytbl_rx_gain_info_2G_rev2_sz;
 
-extern CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_5G_rev2[];
-extern CONST uint32 dot11lcnphytbl_rx_gain_info_5G_rev2_sz;
+extern const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_5G_rev2[];
+extern const uint32 dot11lcnphytbl_rx_gain_info_5G_rev2_sz;
 
-extern CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_extlna_2G_rev2[];
+extern const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_extlna_2G_rev2[];
 
-extern CONST dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_extlna_5G_rev2[];
+extern const dot11lcnphytbl_info_t dot11lcnphytbl_rx_gain_info_extlna_5G_rev2[];
 
 typedef struct {
 	unsigned char gm;
@@ -43,7 +43,7 @@ typedef struct {
 	unsigned char bb_mult;
 } lcnphy_tx_gain_tbl_entry;
 
-extern CONST lcnphy_tx_gain_tbl_entry dot11lcnphy_2GHz_gaintable_rev0[];
-extern CONST lcnphy_tx_gain_tbl_entry dot11lcnphy_2GHz_extPA_gaintable_rev0[];
+extern const lcnphy_tx_gain_tbl_entry dot11lcnphy_2GHz_gaintable_rev0[];
+extern const lcnphy_tx_gain_tbl_entry dot11lcnphy_2GHz_extPA_gaintable_rev0[];
 
-extern CONST lcnphy_tx_gain_tbl_entry dot11lcnphy_5GHz_gaintable_rev0[];
+extern const lcnphy_tx_gain_tbl_entry dot11lcnphy_5GHz_gaintable_rev0[];
--- a/drivers/staging/brcm80211/phy/wlc_phytbl_n.c
+++ b/drivers/staging/brcm80211/phy/wlc_phytbl_n.c
@@ -19,7 +19,7 @@
 #include <wlc_phy_int.h>
 #include <wlc_phytbl_n.h>
 
-CONST uint32 frame_struct_rev0[] = {
+const uint32 frame_struct_rev0[] = {
 	0x08004a04,
 	0x00100000,
 	0x01000a05,
@@ -854,7 +854,7 @@ CONST uint32 frame_struct_rev0[] = {
 	0x00000000,
 };
 
-CONST u8 frame_lut_rev0[] = {
+const u8 frame_lut_rev0[] = {
 	0x02,
 	0x04,
 	0x14,
@@ -889,7 +889,7 @@ CONST u8 frame_lut_rev0[] = {
 	0x2a,
 };
 
-CONST uint32 tmap_tbl_rev0[] = {
+const uint32 tmap_tbl_rev0[] = {
 	0x8a88aa80,
 	0x8aaaaa8a,
 	0x8a8a8aa8,
@@ -1340,7 +1340,7 @@ CONST uint32 tmap_tbl_rev0[] = {
 	0x00000000,
 };
 
-CONST uint32 tdtrn_tbl_rev0[] = {
+const uint32 tdtrn_tbl_rev0[] = {
 	0x061c061c,
 	0x0050ee68,
 	0xf592fe36,
@@ -2047,7 +2047,7 @@ CONST uint32 tdtrn_tbl_rev0[] = {
 	0x00f006be,
 };
 
-CONST uint32 intlv_tbl_rev0[] = {
+const uint32 intlv_tbl_rev0[] = {
 	0x00802070,
 	0x0671188d,
 	0x0a60192c,
@@ -2057,7 +2057,7 @@ CONST uint32 intlv_tbl_rev0[] = {
 	0x00000070,
 };
 
-CONST uint16 pilot_tbl_rev0[] = {
+const uint16 pilot_tbl_rev0[] = {
 	0xff08,
 	0xff08,
 	0xff08,
@@ -2148,7 +2148,7 @@ CONST uint16 pilot_tbl_rev0[] = {
 	0xffff,
 };
 
-CONST uint32 pltlut_tbl_rev0[] = {
+const uint32 pltlut_tbl_rev0[] = {
 	0x76540123,
 	0x62407351,
 	0x76543201,
@@ -2157,7 +2157,7 @@ CONST uint32 pltlut_tbl_rev0[] = {
 	0x76430521,
 };
 
-CONST uint32 tdi_tbl20_ant0_rev0[] = {
+const uint32 tdi_tbl20_ant0_rev0[] = {
 	0x00091226,
 	0x000a1429,
 	0x000b56ad,
@@ -2215,7 +2215,7 @@ CONST uint32 tdi_tbl20_ant0_rev0[] = {
 	0x00000000,
 };
 
-CONST uint32 tdi_tbl20_ant1_rev0[] = {
+const uint32 tdi_tbl20_ant1_rev0[] = {
 	0x00014b26,
 	0x00028d29,
 	0x000393ad,
@@ -2273,7 +2273,7 @@ CONST uint32 tdi_tbl20_ant1_rev0[] = {
 	0x00000000,
 };
 
-CONST uint32 tdi_tbl40_ant0_rev0[] = {
+const uint32 tdi_tbl40_ant0_rev0[] = {
 	0x0011a346,
 	0x00136ccf,
 	0x0014f5d9,
@@ -2386,7 +2386,7 @@ CONST uint32 tdi_tbl40_ant0_rev0[] = {
 	0x00000000,
 };
 
-CONST uint32 tdi_tbl40_ant1_rev0[] = {
+const uint32 tdi_tbl40_ant1_rev0[] = {
 	0x001edb36,
 	0x000129ca,
 	0x0002b353,
@@ -2499,7 +2499,7 @@ CONST uint32 tdi_tbl40_ant1_rev0[] = {
 	0x00000000,
 };
 
-CONST uint16 bdi_tbl_rev0[] = {
+const uint16 bdi_tbl_rev0[] = {
 	0x0070,
 	0x0126,
 	0x012c,
@@ -2508,7 +2508,7 @@ CONST uint16 bdi_tbl_rev0[] = {
 	0x04d2,
 };
 
-CONST uint32 chanest_tbl_rev0[] = {
+const uint32 chanest_tbl_rev0[] = {
 	0x44444444,
 	0x44444444,
 	0x44444444,
@@ -2607,7 +2607,7 @@ CONST uint32 chanest_tbl_rev0[] = {
 	0x10101010,
 };
 
-CONST u8 mcs_tbl_rev0[] = {
+const u8 mcs_tbl_rev0[] = {
 	0x00,
 	0x08,
 	0x0a,
@@ -2738,7 +2738,7 @@ CONST u8 mcs_tbl_rev0[] = {
 	0x00,
 };
 
-CONST uint32 noise_var_tbl0_rev0[] = {
+const uint32 noise_var_tbl0_rev0[] = {
 	0x020c020c,
 	0x0000014d,
 	0x020c020c,
@@ -2997,7 +2997,7 @@ CONST uint32 noise_var_tbl0_rev0[] = {
 	0x0000014d,
 };
 
-CONST uint32 noise_var_tbl1_rev0[] = {
+const uint32 noise_var_tbl1_rev0[] = {
 	0x020c020c,
 	0x0000014d,
 	0x020c020c,
@@ -3256,7 +3256,7 @@ CONST uint32 noise_var_tbl1_rev0[] = {
 	0x0000014d,
 };
 
-CONST u8 est_pwr_lut_core0_rev0[] = {
+const u8 est_pwr_lut_core0_rev0[] = {
 	0x50,
 	0x4f,
 	0x4e,
@@ -3323,7 +3323,7 @@ CONST u8 est_pwr_lut_core0_rev0[] = {
 	0x11,
 };
 
-CONST u8 est_pwr_lut_core1_rev0[] = {
+const u8 est_pwr_lut_core1_rev0[] = {
 	0x50,
 	0x4f,
 	0x4e,
@@ -3390,7 +3390,7 @@ CONST u8 est_pwr_lut_core1_rev0[] = {
 	0x11,
 };
 
-CONST u8 adj_pwr_lut_core0_rev0[] = {
+const u8 adj_pwr_lut_core0_rev0[] = {
 	0x00,
 	0x00,
 	0x00,
@@ -3521,7 +3521,7 @@ CONST u8 adj_pwr_lut_core0_rev0[] = {
 	0x00,
 };
 
-CONST u8 adj_pwr_lut_core1_rev0[] = {
+const u8 adj_pwr_lut_core1_rev0[] = {
 	0x00,
 	0x00,
 	0x00,
@@ -3652,7 +3652,7 @@ CONST u8 adj_pwr_lut_core1_rev0[] = {
 	0x00,
 };
 
-CONST uint32 gainctrl_lut_core0_rev0[] = {
+const uint32 gainctrl_lut_core0_rev0[] = {
 	0x03cc2b44,
 	0x03cc2b42,
 	0x03cc2b40,
@@ -3783,7 +3783,7 @@ CONST uint32 gainctrl_lut_core0_rev0[] =
 	0x00002b00,
 };
 
-CONST uint32 gainctrl_lut_core1_rev0[] = {
+const uint32 gainctrl_lut_core1_rev0[] = {
 	0x03cc2b44,
 	0x03cc2b42,
 	0x03cc2b40,
@@ -3914,7 +3914,7 @@ CONST uint32 gainctrl_lut_core1_rev0[] =
 	0x00002b00,
 };
 
-CONST uint32 iq_lut_core0_rev0[] = {
+const uint32 iq_lut_core0_rev0[] = {
 	0x0000007f,
 	0x0000007f,
 	0x0000007f,
@@ -4045,7 +4045,7 @@ CONST uint32 iq_lut_core0_rev0[] = {
 	0x0000007f,
 };
 
-CONST uint32 iq_lut_core1_rev0[] = {
+const uint32 iq_lut_core1_rev0[] = {
 	0x0000007f,
 	0x0000007f,
 	0x0000007f,
@@ -4176,7 +4176,7 @@ CONST uint32 iq_lut_core1_rev0[] = {
 	0x0000007f,
 };
 
-CONST uint16 loft_lut_core0_rev0[] = {
+const uint16 loft_lut_core0_rev0[] = {
 	0x0000,
 	0x0101,
 	0x0002,
@@ -4307,7 +4307,7 @@ CONST uint16 loft_lut_core0_rev0[] = {
 	0x0103,
 };
 
-CONST uint16 loft_lut_core1_rev0[] = {
+const uint16 loft_lut_core1_rev0[] = {
 	0x0000,
 	0x0101,
 	0x0002,
@@ -4438,7 +4438,7 @@ CONST uint16 loft_lut_core1_rev0[] = {
 	0x0103,
 };
 
-CONST mimophytbl_info_t mimophytbl_info_rev0_volatile[] = {
+const mimophytbl_info_t mimophytbl_info_rev0_volatile[] = {
 	{&bdi_tbl_rev0, sizeof(bdi_tbl_rev0) / sizeof(bdi_tbl_rev0[0]), 21, 0,
 	 16}
 	,
@@ -4486,7 +4486,7 @@ CONST mimophytbl_info_t mimophytbl_info_
 	,
 };
 
-CONST mimophytbl_info_t mimophytbl_info_rev0[] = {
+const mimophytbl_info_t mimophytbl_info_rev0[] = {
 	{&frame_struct_rev0,
 	 sizeof(frame_struct_rev0) / sizeof(frame_struct_rev0[0]), 10, 0, 32}
 	,
@@ -4536,13 +4536,13 @@ CONST mimophytbl_info_t mimophytbl_info_
 	,
 };
 
-CONST uint32 mimophytbl_info_sz_rev0 =
+const uint32 mimophytbl_info_sz_rev0 =
     sizeof(mimophytbl_info_rev0) / sizeof(mimophytbl_info_rev0[0]);
-CONST uint32 mimophytbl_info_sz_rev0_volatile =
+const uint32 mimophytbl_info_sz_rev0_volatile =
     sizeof(mimophytbl_info_rev0_volatile) /
     sizeof(mimophytbl_info_rev0_volatile[0]);
 
-CONST uint16 ant_swctrl_tbl_rev3[] = {
+const uint16 ant_swctrl_tbl_rev3[] = {
 	0x0082,
 	0x0082,
 	0x0211,
@@ -4577,7 +4577,7 @@ CONST uint16 ant_swctrl_tbl_rev3[] = {
 	0x0000,
 };
 
-CONST uint16 ant_swctrl_tbl_rev3_1[] = {
+const uint16 ant_swctrl_tbl_rev3_1[] = {
 	0x0022,
 	0x0022,
 	0x0011,
@@ -4612,7 +4612,7 @@ CONST uint16 ant_swctrl_tbl_rev3_1[] = {
 	0x0000,
 };
 
-CONST uint16 ant_swctrl_tbl_rev3_2[] = {
+const uint16 ant_swctrl_tbl_rev3_2[] = {
 	0x0088,
 	0x0088,
 	0x0044,
@@ -4647,7 +4647,7 @@ CONST uint16 ant_swctrl_tbl_rev3_2[] = {
 	0x0000,
 };
 
-CONST uint16 ant_swctrl_tbl_rev3_3[] = {
+const uint16 ant_swctrl_tbl_rev3_3[] = {
 	0x022,
 	0x022,
 	0x011,
@@ -4682,7 +4682,7 @@ CONST uint16 ant_swctrl_tbl_rev3_3[] = {
 	0x3cc
 };
 
-CONST uint32 frame_struct_rev3[] = {
+const uint32 frame_struct_rev3[] = {
 	0x08004a04,
 	0x00100000,
 	0x01000a05,
@@ -5517,7 +5517,7 @@ CONST uint32 frame_struct_rev3[] = {
 	0x00000000,
 };
 
-CONST uint16 pilot_tbl_rev3[] = {
+const uint16 pilot_tbl_rev3[] = {
 	0xff08,
 	0xff08,
 	0xff08,
@@ -5608,7 +5608,7 @@ CONST uint16 pilot_tbl_rev3[] = {
 	0xffff,
 };
 
-CONST uint32 tmap_tbl_rev3[] = {
+const uint32 tmap_tbl_rev3[] = {
 	0x8a88aa80,
 	0x8aaaaa8a,
 	0x8a8a8aa8,
@@ -6059,7 +6059,7 @@ CONST uint32 tmap_tbl_rev3[] = {
 	0x00000000,
 };
 
-CONST uint32 intlv_tbl_rev3[] = {
+const uint32 intlv_tbl_rev3[] = {
 	0x00802070,
 	0x0671188d,
 	0x0a60192c,
@@ -6069,7 +6069,7 @@ CONST uint32 intlv_tbl_rev3[] = {
 	0x00000070,
 };
 
-CONST uint32 tdtrn_tbl_rev3[] = {
+const uint32 tdtrn_tbl_rev3[] = {
 	0x061c061c,
 	0x0050ee68,
 	0xf592fe36,
@@ -6776,7 +6776,7 @@ CONST uint32 tdtrn_tbl_rev3[] = {
 	0x00f006be,
 };
 
-CONST uint32 noise_var_tbl_rev3[] = {
+const uint32 noise_var_tbl_rev3[] = {
 	0x02110211,
 	0x0000014d,
 	0x02110211,
@@ -7035,7 +7035,7 @@ CONST uint32 noise_var_tbl_rev3[] = {
 	0x0000014d,
 };
 
-CONST uint16 mcs_tbl_rev3[] = {
+const uint16 mcs_tbl_rev3[] = {
 	0x0000,
 	0x0008,
 	0x000a,
@@ -7166,7 +7166,7 @@ CONST uint16 mcs_tbl_rev3[] = {
 	0x0007,
 };
 
-CONST uint32 tdi_tbl20_ant0_rev3[] = {
+const uint32 tdi_tbl20_ant0_rev3[] = {
 	0x00091226,
 	0x000a1429,
 	0x000b56ad,
@@ -7224,7 +7224,7 @@ CONST uint32 tdi_tbl20_ant0_rev3[] = {
 	0x00000000,
 };
 
-CONST uint32 tdi_tbl20_ant1_rev3[] = {
+const uint32 tdi_tbl20_ant1_rev3[] = {
 	0x00014b26,
 	0x00028d29,
 	0x000393ad,
@@ -7282,7 +7282,7 @@ CONST uint32 tdi_tbl20_ant1_rev3[] = {
 	0x00000000,
 };
 
-CONST uint32 tdi_tbl40_ant0_rev3[] = {
+const uint32 tdi_tbl40_ant0_rev3[] = {
 	0x0011a346,
 	0x00136ccf,
 	0x0014f5d9,
@@ -7395,7 +7395,7 @@ CONST uint32 tdi_tbl40_ant0_rev3[] = {
 	0x00000000,
 };
 
-CONST uint32 tdi_tbl40_ant1_rev3[] = {
+const uint32 tdi_tbl40_ant1_rev3[] = {
 	0x001edb36,
 	0x000129ca,
 	0x0002b353,
@@ -7508,7 +7508,7 @@ CONST uint32 tdi_tbl40_ant1_rev3[] = {
 	0x00000000,
 };
 
-CONST uint32 pltlut_tbl_rev3[] = {
+const uint32 pltlut_tbl_rev3[] = {
 	0x76540213,
 	0x62407351,
 	0x76543210,
@@ -7517,7 +7517,7 @@ CONST uint32 pltlut_tbl_rev3[] = {
 	0x76430521,
 };
 
-CONST uint32 chanest_tbl_rev3[] = {
+const uint32 chanest_tbl_rev3[] = {
 	0x44444444,
 	0x44444444,
 	0x44444444,
@@ -7616,7 +7616,7 @@ CONST uint32 chanest_tbl_rev3[] = {
 	0x10101010,
 };
 
-CONST u8 frame_lut_rev3[] = {
+const u8 frame_lut_rev3[] = {
 	0x02,
 	0x04,
 	0x14,
@@ -7651,7 +7651,7 @@ CONST u8 frame_lut_rev3[] = {
 	0x2a,
 };
 
-CONST u8 est_pwr_lut_core0_rev3[] = {
+const u8 est_pwr_lut_core0_rev3[] = {
 	0x55,
 	0x54,
 	0x54,
@@ -7718,7 +7718,7 @@ CONST u8 est_pwr_lut_core0_rev3[] = {
 	0xfd,
 };
 
-CONST u8 est_pwr_lut_core1_rev3[] = {
+const u8 est_pwr_lut_core1_rev3[] = {
 	0x55,
 	0x54,
 	0x54,
@@ -7785,7 +7785,7 @@ CONST u8 est_pwr_lut_core1_rev3[] = {
 	0xfd,
 };
 
-CONST u8 adj_pwr_lut_core0_rev3[] = {
+const u8 adj_pwr_lut_core0_rev3[] = {
 	0x00,
 	0x00,
 	0x00,
@@ -7916,7 +7916,7 @@ CONST u8 adj_pwr_lut_core0_rev3[] = {
 	0x00,
 };
 
-CONST u8 adj_pwr_lut_core1_rev3[] = {
+const u8 adj_pwr_lut_core1_rev3[] = {
 	0x00,
 	0x00,
 	0x00,
@@ -8047,7 +8047,7 @@ CONST u8 adj_pwr_lut_core1_rev3[] = {
 	0x00,
 };
 
-CONST uint32 gainctrl_lut_core0_rev3[] = {
+const uint32 gainctrl_lut_core0_rev3[] = {
 	0x5bf70044,
 	0x5bf70042,
 	0x5bf70040,
@@ -8178,7 +8178,7 @@ CONST uint32 gainctrl_lut_core0_rev3[] =
 	0x5b07001c,
 };
 
-CONST uint32 gainctrl_lut_core1_rev3[] = {
+const uint32 gainctrl_lut_core1_rev3[] = {
 	0x5bf70044,
 	0x5bf70042,
 	0x5bf70040,
@@ -8309,7 +8309,7 @@ CONST uint32 gainctrl_lut_core1_rev3[] =
 	0x5b07001c,
 };
 
-CONST uint32 iq_lut_core0_rev3[] = {
+const uint32 iq_lut_core0_rev3[] = {
 	0x00000000,
 	0x00000000,
 	0x00000000,
@@ -8440,7 +8440,7 @@ CONST uint32 iq_lut_core0_rev3[] = {
 	0x00000000,
 };
 
-CONST uint32 iq_lut_core1_rev3[] = {
+const uint32 iq_lut_core1_rev3[] = {
 	0x00000000,
 	0x00000000,
 	0x00000000,
@@ -8571,7 +8571,7 @@ CONST uint32 iq_lut_core1_rev3[] = {
 	0x00000000,
 };
 
-CONST uint16 loft_lut_core0_rev3[] = {
+const uint16 loft_lut_core0_rev3[] = {
 	0x0000,
 	0x0000,
 	0x0000,
@@ -8702,7 +8702,7 @@ CONST uint16 loft_lut_core0_rev3[] = {
 	0x0000,
 };
 
-CONST uint16 loft_lut_core1_rev3[] = {
+const uint16 loft_lut_core1_rev3[] = {
 	0x0000,
 	0x0000,
 	0x0000,
@@ -8833,7 +8833,7 @@ CONST uint16 loft_lut_core1_rev3[] = {
 	0x0000,
 };
 
-CONST uint16 papd_comp_rfpwr_tbl_core0_rev3[] = {
+const uint16 papd_comp_rfpwr_tbl_core0_rev3[] = {
 	0x0036,
 	0x0036,
 	0x0036,
@@ -8964,7 +8964,7 @@ CONST uint16 papd_comp_rfpwr_tbl_core0_r
 	0x01d6,
 };
 
-CONST uint16 papd_comp_rfpwr_tbl_core1_rev3[] = {
+const uint16 papd_comp_rfpwr_tbl_core1_rev3[] = {
 	0x0036,
 	0x0036,
 	0x0036,
@@ -9095,7 +9095,7 @@ CONST uint16 papd_comp_rfpwr_tbl_core1_r
 	0x01d6,
 };
 
-CONST uint32 papd_comp_epsilon_tbl_core0_rev3[] = {
+const uint32 papd_comp_epsilon_tbl_core0_rev3[] = {
 	0x00000000,
 	0x00001fa0,
 	0x00019f78,
@@ -9162,7 +9162,7 @@ CONST uint32 papd_comp_epsilon_tbl_core0
 	0x03e38ffe,
 };
 
-CONST uint32 papd_cal_scalars_tbl_core0_rev3[] = {
+const uint32 papd_cal_scalars_tbl_core0_rev3[] = {
 	0x05af005a,
 	0x0571005e,
 	0x05040066,
@@ -9229,7 +9229,7 @@ CONST uint32 papd_cal_scalars_tbl_core0_
 	0x002606a4,
 };
 
-CONST uint32 papd_comp_epsilon_tbl_core1_rev3[] = {
+const uint32 papd_comp_epsilon_tbl_core1_rev3[] = {
 	0x00000000,
 	0x00001fa0,
 	0x00019f78,
@@ -9296,7 +9296,7 @@ CONST uint32 papd_comp_epsilon_tbl_core1
 	0x03e38ffe,
 };
 
-CONST uint32 papd_cal_scalars_tbl_core1_rev3[] = {
+const uint32 papd_cal_scalars_tbl_core1_rev3[] = {
 	0x05af005a,
 	0x0571005e,
 	0x05040066,
@@ -9363,34 +9363,34 @@ CONST uint32 papd_cal_scalars_tbl_core1_
 	0x002606a4,
 };
 
-CONST mimophytbl_info_t mimophytbl_info_rev3_volatile[] = {
+const mimophytbl_info_t mimophytbl_info_rev3_volatile[] = {
 	{&ant_swctrl_tbl_rev3,
 	 sizeof(ant_swctrl_tbl_rev3) / sizeof(ant_swctrl_tbl_rev3[0]), 9, 0, 16}
 	,
 };
 
-CONST mimophytbl_info_t mimophytbl_info_rev3_volatile1[] = {
+const mimophytbl_info_t mimophytbl_info_rev3_volatile1[] = {
 	{&ant_swctrl_tbl_rev3_1,
 	 sizeof(ant_swctrl_tbl_rev3_1) / sizeof(ant_swctrl_tbl_rev3_1[0]), 9, 0,
 	 16}
 	,
 };
 
-CONST mimophytbl_info_t mimophytbl_info_rev3_volatile2[] = {
+const mimophytbl_info_t mimophytbl_info_rev3_volatile2[] = {
 	{&ant_swctrl_tbl_rev3_2,
 	 sizeof(ant_swctrl_tbl_rev3_2) / sizeof(ant_swctrl_tbl_rev3_2[0]), 9, 0,
 	 16}
 	,
 };
 
-CONST mimophytbl_info_t mimophytbl_info_rev3_volatile3[] = {
+const mimophytbl_info_t mimophytbl_info_rev3_volatile3[] = {
 	{&ant_swctrl_tbl_rev3_3,
 	 sizeof(ant_swctrl_tbl_rev3_3) / sizeof(ant_swctrl_tbl_rev3_3[0]), 9, 0,
 	 16}
 	,
 };
 
-CONST mimophytbl_info_t mimophytbl_info_rev3[] = {
+const mimophytbl_info_t mimophytbl_info_rev3[] = {
 	{&frame_struct_rev3,
 	 sizeof(frame_struct_rev3) / sizeof(frame_struct_rev3[0]), 10, 0, 32}
 	,
@@ -9476,22 +9476,22 @@ CONST mimophytbl_info_t mimophytbl_info_
 	 16}
 };
 
-CONST uint32 mimophytbl_info_sz_rev3 =
+const uint32 mimophytbl_info_sz_rev3 =
     sizeof(mimophytbl_info_rev3) / sizeof(mimophytbl_info_rev3[0]);
-CONST uint32 mimophytbl_info_sz_rev3_volatile =
+const uint32 mimophytbl_info_sz_rev3_volatile =
     sizeof(mimophytbl_info_rev3_volatile) /
     sizeof(mimophytbl_info_rev3_volatile[0]);
-CONST uint32 mimophytbl_info_sz_rev3_volatile1 =
+const uint32 mimophytbl_info_sz_rev3_volatile1 =
     sizeof(mimophytbl_info_rev3_volatile1) /
     sizeof(mimophytbl_info_rev3_volatile1[0]);
-CONST uint32 mimophytbl_info_sz_rev3_volatile2 =
+const uint32 mimophytbl_info_sz_rev3_volatile2 =
     sizeof(mimophytbl_info_rev3_volatile2) /
     sizeof(mimophytbl_info_rev3_volatile2[0]);
-CONST uint32 mimophytbl_info_sz_rev3_volatile3 =
+const uint32 mimophytbl_info_sz_rev3_volatile3 =
     sizeof(mimophytbl_info_rev3_volatile3) /
     sizeof(mimophytbl_info_rev3_volatile3[0]);
 
-CONST uint32 tmap_tbl_rev7[] = {
+const uint32 tmap_tbl_rev7[] = {
 	0x8a88aa80,
 	0x8aaaaa8a,
 	0x8a8a8aa8,
@@ -9942,7 +9942,7 @@ CONST uint32 tmap_tbl_rev7[] = {
 	0x00000000,
 };
 
-CONST uint32 noise_var_tbl_rev7[] = {
+const uint32 noise_var_tbl_rev7[] = {
 	0x020c020c,
 	0x0000014d,
 	0x020c020c,
@@ -10201,7 +10201,7 @@ CONST uint32 noise_var_tbl_rev7[] = {
 	0x0000014d,
 };
 
-CONST uint32 papd_comp_epsilon_tbl_core0_rev7[] = {
+const uint32 papd_comp_epsilon_tbl_core0_rev7[] = {
 	0x00000000,
 	0x00000000,
 	0x00016023,
@@ -10268,7 +10268,7 @@ CONST uint32 papd_comp_epsilon_tbl_core0
 	0x0156cfff,
 };
 
-CONST uint32 papd_cal_scalars_tbl_core0_rev7[] = {
+const uint32 papd_cal_scalars_tbl_core0_rev7[] = {
 	0x0b5e002d,
 	0x0ae2002f,
 	0x0a3b0032,
@@ -10335,7 +10335,7 @@ CONST uint32 papd_cal_scalars_tbl_core0_
 	0x004e068c,
 };
 
-CONST uint32 papd_comp_epsilon_tbl_core1_rev7[] = {
+const uint32 papd_comp_epsilon_tbl_core1_rev7[] = {
 	0x00000000,
 	0x00000000,
 	0x00016023,
@@ -10402,7 +10402,7 @@ CONST uint32 papd_comp_epsilon_tbl_core1
 	0x0156cfff,
 };
 
-CONST uint32 papd_cal_scalars_tbl_core1_rev7[] = {
+const uint32 papd_cal_scalars_tbl_core1_rev7[] = {
 	0x0b5e002d,
 	0x0ae2002f,
 	0x0a3b0032,
@@ -10469,7 +10469,7 @@ CONST uint32 papd_cal_scalars_tbl_core1_
 	0x004e068c,
 };
 
-CONST mimophytbl_info_t mimophytbl_info_rev7[] = {
+const mimophytbl_info_t mimophytbl_info_rev7[] = {
 	{&frame_struct_rev3,
 	 sizeof(frame_struct_rev3) / sizeof(frame_struct_rev3[0]), 10, 0, 32}
 	,
@@ -10580,10 +10580,10 @@ CONST mimophytbl_info_t mimophytbl_info_
 	,
 };
 
-CONST uint32 mimophytbl_info_sz_rev7 =
+const uint32 mimophytbl_info_sz_rev7 =
     sizeof(mimophytbl_info_rev7) / sizeof(mimophytbl_info_rev7[0]);
 
-CONST mimophytbl_info_t mimophytbl_info_rev16[] = {
+const mimophytbl_info_t mimophytbl_info_rev16[] = {
 	{&noise_var_tbl_rev7,
 	 sizeof(noise_var_tbl_rev7) / sizeof(noise_var_tbl_rev7[0]), 16, 0, 32}
 	,
@@ -10627,5 +10627,5 @@ CONST mimophytbl_info_t mimophytbl_info_
 	,
 };
 
-CONST uint32 mimophytbl_info_sz_rev16 =
+const uint32 mimophytbl_info_sz_rev16 =
     sizeof(mimophytbl_info_rev16) / sizeof(mimophytbl_info_rev16[0]);
--- a/drivers/staging/brcm80211/phy/wlc_phytbl_n.h
+++ b/drivers/staging/brcm80211/phy/wlc_phytbl_n.h
@@ -18,22 +18,22 @@
 
 typedef phytbl_info_t mimophytbl_info_t;
 
-extern CONST mimophytbl_info_t mimophytbl_info_rev0[],
+extern const mimophytbl_info_t mimophytbl_info_rev0[],
     mimophytbl_info_rev0_volatile[];
-extern CONST uint32 mimophytbl_info_sz_rev0, mimophytbl_info_sz_rev0_volatile;
+extern const uint32 mimophytbl_info_sz_rev0, mimophytbl_info_sz_rev0_volatile;
 
-extern CONST mimophytbl_info_t mimophytbl_info_rev3[],
+extern const mimophytbl_info_t mimophytbl_info_rev3[],
     mimophytbl_info_rev3_volatile[], mimophytbl_info_rev3_volatile1[],
     mimophytbl_info_rev3_volatile2[], mimophytbl_info_rev3_volatile3[];
-extern CONST uint32 mimophytbl_info_sz_rev3, mimophytbl_info_sz_rev3_volatile,
+extern const uint32 mimophytbl_info_sz_rev3, mimophytbl_info_sz_rev3_volatile,
     mimophytbl_info_sz_rev3_volatile1, mimophytbl_info_sz_rev3_volatile2,
     mimophytbl_info_sz_rev3_volatile3;
 
-extern CONST uint32 noise_var_tbl_rev3[];
+extern const uint32 noise_var_tbl_rev3[];
 
-extern CONST mimophytbl_info_t mimophytbl_info_rev7[];
-extern CONST uint32 mimophytbl_info_sz_rev7;
-extern CONST uint32 noise_var_tbl_rev7[];
+extern const mimophytbl_info_t mimophytbl_info_rev7[];
+extern const uint32 mimophytbl_info_sz_rev7;
+extern const uint32 noise_var_tbl_rev7[];
 
-extern CONST mimophytbl_info_t mimophytbl_info_rev16[];
-extern CONST uint32 mimophytbl_info_sz_rev16;
+extern const mimophytbl_info_t mimophytbl_info_rev16[];
+extern const uint32 mimophytbl_info_sz_rev16;
